MODULE.bazel

```diff
@@ -23,7 +23,7 @@ bazel_dep(name = "glog", version = "0.7.1")
 bazel_dep(name = "googletest", version = "1.15.2")
 bazel_dep(name = "lz4", version = "1.9.4")
 bazel_dep(name = "platforms", version = "0.0.10")
-bazel_dep(name = "rules_cc", version = "0.1.0")
+bazel_dep(name = "rules_cc", version = "0.0.17")
```

Comment:
This seems relevant for this PR, other changes should be in a separate PR.

Reply:
The other changes are required to this pass. If you want this change as a single commit, we would have to bump all boost first in a seperate PR.

Reply:
I just now see that this is a downgrade. Why do we need to downgrade?

Reply:
0.1.0 was yanked on BCR. But a few days ago 0.1.1 was released so upgrading to 0.1.1 is also possible.